Hakawai/Core/HKWTextView.h

```diff
@@ -41,6 +41,9 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @interface HKWTextView : UITextView
 
++ (BOOL) enableExperimentalDeadLockFix;
```

Comment:
You can just use a class property rather than having both of these funcs and the static var